projects/plugins/jetpack/tests/php/general/test-class.jetpack-gutenberg.php

```diff
@@ -13,12 +13,12 @@ public function set_up() {
 		parent::set_up();
 		if ( ! function_exists( 'register_block_type' ) ) {
 			$this->markTestSkipped( 'register_block_type not available' );
-			return;
+			return; // @phan-suppress-current-line PhanPluginUnreachableCode
 		}
 
 		if ( ! class_exists( 'WP_Block_Type_Registry' ) ) {
 			$this->markTestSkipped( 'WP_Block_Type_Registry not available' );
-			return;
+			return; // @phan-suppress-current-line PhanPluginUnreachableCode
```

Comment:
Probably shouldn't have the `return` at all here.

Reply:
Same.